wordpress-mobile/gutenberg-mobile
# Remove BlockModel var and prop.

This is no longer needed due to changes made in Aztec that update the
typing properties correctly.

To test:
 - Start the demo app
 - Add a Heading block
 - Change the header type
 - See if text and placeholder update correctly

PR submission checklist:
